tests/system/action/user/test_create.py

```diff
@@ -199,3 +208,320 @@ def test_create_empty_username(self) -> None:
             "data.username must be longer than or equal to 1 characters",
             response.json["message"],
         )
+
+    def test_create_permission_nothing(self) -> None:
+        self.permission_setup()
+        response = self.request(
+            "user.create",
+            {
+                "username": "username",
+                "vote_weight_$": {1: "1.000000"},
+                "group_$_ids": {1: [1]},
+            },
+        )
+        self.assert_status_code(response, 403)
+        self.assertIn(
+            "or alternative {'Committee Manager Right for meetings {1}'}.",
```

Comment:
Yes, the whole message is sometimes `"You are not allowed to perform action user.update. Missing permissions {'OrganisationManagementLevel.CAN_MANAGE_USERS', '_User.CAN_MANAGE for meeting 1'} or alternative {'Committee Manager Right for meetings {1}'}."`. Problem in catching this assert is, that the sequence of the errors changes because of unsorted dict (fields of instance).

Reply:
If the order is the problem, how about multiple `assertIn` calls, one for each missing permission?